internal/configuration/validator/authentication.go

```diff
@@ -120,6 +120,8 @@ func validateLDAPAuthenticationBackend(configuration *schema.LDAPAuthenticationB
 		setDefaultImplementationCustomLDAPAuthenticationBackend(configuration)
 	case schema.LDAPImplementationActiveDirectory:
 		setDefaultImplementationActiveDirectoryLDAPAuthenticationBackend(configuration)
+	case schema.LDAPImplementationFreeIPA:
+		setDefaultImplementationFreeIPALDAPAuthenticationBackend(configuration)
 	default:
 		validator.Push(fmt.Errorf("authentication backend ldap implementation must be blank or one of the following values `%s`, `%s`", schema.LDAPImplementationCustom, schema.LDAPImplementationActiveDirectory))
```

Comment:
Add this option too `schema.LDAPImplementationFreeIPA`

Reply:
Sorry it's for the line 126

Reply:
Nice spot!